src/keys.c

```diff
@@ -3559,7 +3559,7 @@ int SetKeysSide(WOLFSSL* ssl, enum encrypt_side side)
             cbInfo->side = side;
         #elif defined(WOLFSSL_RENESAS_TSIP_TLS)
             TsipUserCtx* cbInfo = (TsipUserCtx*)ctx;
-            cbInfo->key_side = side;
+            _ACCESSOR(cbInfo)->key_side = side;
```

Comment:
From the .c side it seems like using _ACCESSOR() is just not needed. Why not reference the internal directly and avoid the macro? If a macro is needed please use something more specific like WC_WSIP_STUCT

Reply:
There was no specific reason. Just a personal preference that I thought avoiding multiple arrow operators, increase readability.